common/lib/xmodule/xmodule/js/src/video/09_events_plugin.js

```diff
@@ -94,6 +94,7 @@
                     new_time: time,
                     type: type
                 });
+                this.emitPlayVideoEvent = true;
```

Comment:
yes, it will generate extra events. Whenever video will play after buffering, it will log a `play_video` event. `this.emitPlayVideoEvent = false;` will restrict it.